CODE_OF_CONDUCT.md

```diff
@@ -20,11 +20,13 @@ If you are being harassed, notice that someone else is being harassed, or have a
 
 We expect everyone to follow these rules anywhere in the CocoaPods project’s codebases, issue trackers, IRC channel, group chat, and mailing lists.
 
+This code of conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
```

Comment:
Thanks for taking the time to provide this change :+1:

I have a question, though, that I hope you can answer: the text uses “when”, which to me seems to mean _only at times when actively representing the project_, whereas others think that a core contributor is _always_ implicitly representing the project and thus it means _always_. How is this meant?

Regardless of how that original text was meant, I propose that we change it that this _always_ applies to [‘core contributors’](https://github.com/orgs/CocoaPods/teams/core) and others _only_ when they are actively representing the project. Asking (very) occasional contributors to walk the fine line of what defines a public space seems too ambiguous to me, which means we can’t reliably enforce it nor would it make people feel confident about contributing/voicing their opinions.